apps/hash-frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card.tsx

```diff
@@ -386,21 +391,25 @@ const PropertyRow = ({
 
 export const PropertyTypeRow = ({
   propertyIndex,
+  property,
+  propertyId,
   onRemove,
   onUpdateVersion,
+  flash,
 }: {
   propertyIndex: number;
+  property: PropertyType | undefined;
+  propertyId: VersionedUri;
   onRemove: () => void;
   onUpdateVersion: (nextId: VersionedUri) => void;
+  flash: boolean;
 }) => {
   const { control } = useFormContext<EntityTypeEditorForm>();
 
-  const [$id, array] = useWatch({
+  // @todo move down
```

Comment:
What does this `@todo` mean? Can it happen now, or be better described for later?

Reply:
Apologies, I thought I'd updated all the comments. This variable is only used in a lower component, and subscribing to it at the lowest possible level reduces the size of the re-render tree when its changed. It's a principle of react-hook-form that you should isolate your re-renders. I'll fix it.

Reply:
Sorry, I just remembered why I didn't make this change. Because of the property object recursiveness, the logic for working out if is array is not the same for every instance of `PropertyRow`, so we can't move this down. I meant to remove this comment. Doing that now.